mmv1/products/notebooks/api.yaml

```
@@ -281,6 +281,39 @@ objects:
               Defines whether the instance has the vTPM enabled.
               Enabled by default.
             default_value: true
+      - !ruby/object:Api::Type::Enum
+        name: 'nicType'
+        description: |
+          The type of vNIC driver.
+        values:
+          - UNSPECIFIED_NIC_TYPE
+          - VIRTIO_NET
+          - GVNIC
+        min_version: beta
+      - !ruby/object:Api::Type::NestedObject
+        name: 'reservationAffinity'
+        description: |
+          Reservation Affinity for consuming Zonal reservation.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: 'consume_reservation_type'
```

```suggestion
            name: 'consumeReservationType'
```

This caused us to send the message

```
 "reservationAffinity": {
  "consume_reservation_type": "NO_RESERVATION"
 },
```

instead of

```
 "reservationAffinity": {
  "consumeReservationType": "NO_RESERVATION"
 },
```

and then look for the key `consume_reservation_type` instead of `consumeReservationType` in the response.